src/cc_toolchain/cc_toolchain_config.bzl

```diff
@@ -585,30 +553,29 @@ def _clang_impl(ctx):
         static_libgcc,
         pic_feature,
         supports_pic_feature,
-        stdlib_feature,
         builtin_include_directories_feature,
         common_feature,
+        stdlib_feature,
         lld_feature,
-        clang_warnings_feature,
         coverage_feature,
         opt_feature,
         runtime_library_search_directories,
     ]
 
     tool_paths = [
         tool_path(name = "gcc", path = ctx.attr.host_compiler_path),
-        tool_path(name = "ar", path = ctx.attr.host_compiler_prefix + "/clang-ar"),
-        tool_path(name = "compat-ld", path = ctx.attr.host_compiler_prefix + "/clang-lld"),
-        tool_path(name = "cpp", path = ctx.attr.host_compiler_prefix + "/clang-cpp"),
-        tool_path(name = "dwp", path = ctx.attr.host_compiler_prefix + "/clang-dwp"),
+        tool_path(name = "ar", path = ctx.attr.host_compiler_prefix + "/linux_gcc-ar"),
+        tool_path(name = "compat-ld", path = ctx.attr.host_compiler_prefix + "/linux_gcc-lld"),
+        tool_path(name = "cpp", path = ctx.attr.host_compiler_prefix + "/linux_gcc-cpp"),
+        tool_path(name = "dwp", path = ctx.attr.host_compiler_prefix + "/linux_gcc-dwp"),
         # TODO: note about executable we're using here (ie. not 'llvm-profdata')
         #tool_path(name = "gcov", path = "/usr/bin/gcov"),
-        tool_path(name = "gcov", path = ctx.attr.host_compiler_prefix + "/clang-profdata"),
-        tool_path(name = "ld", path = ctx.attr.host_compiler_prefix + "/clang-lld"),
-        tool_path(name = "nm", path = ctx.attr.host_compiler_prefix + "/clang-nm"),
-        tool_path(name = "objcopy", path = ctx.attr.host_compiler_prefix + "/clang-objcopy"),
-        tool_path(name = "objdump", path = ctx.attr.host_compiler_prefix + "/clang-objdump"),
-        tool_path(name = "strip", path = ctx.attr.host_compiler_prefix + "/clang-strip"),
+        tool_path(name = "gcov", path = ctx.attr.host_compiler_prefix + "/linux_gcc-profdata"),
```

Comment:
Pretty sure this line is the culprit for coverage not working. `linux_gcc-profdata` does not exist. I fixed it by just commenting this line out and using the line above (ie. `/usr/bin/gcov`), but we should probably use a wrapper here to be consistent.

Reply:
I just ran `new_geom/angle_test`, but here's the coverage output: https://codecov.io/gh/UBC-Thunderbots/Software/tree/c41ed5c98b3debdfbace78a42bbbc22e755d7957

Reply:
Thanks for this!